### PR TITLE
Propagate controlsList from amp-video to video

### PR DESCRIPTION
**Pull request by aghassemi**
Closes https://github.com/ampproject/amphtml/issues/11328 
/cc @jessanastasio

### CODE REVIEW

File: extensions/amp-video/validator-amp-video.protoascii [hunk 1]
```diff
@@ -47,6 +47,7 @@ tags: {  # <amp-video>
     name: "controls"
     value: ""
   }
+  attrs: { name: "controlslist" }
```
Comment (cvializ) [inline extensions/amp-video/validator-amp-video.protoascii]: Is this case sensitive? if so, should these be `controlsList`?

Comment (aghassemi) [reply extensions/amp-video/validator-amp-video.protoascii]: is not as I learned from https://github.com/ampproject/amphtml/pull/10976

Comment (cvializ) [reply extensions/amp-video/validator-amp-video.protoascii]: huh nice!

File: extensions/amp-video/0.1/test/test-amp-video.js [hunk 2]
```diff
@@ -165,13 +165,14 @@ describes.realWin('amp-video', {
...
 
-  it('should set poster and controls in prerender mode', () => {
+  it('should set poster, controls, controlsList in prerender mode', () => {
     return getVideo({
       src: 'video.mp4',
       width: 160,
       height: 90,
       'poster': 'img.png',
       'controls': '',
+      'controlsList': 'nofullscreen nodownload noremote',
```
Comment (wassgha) [inline extensions/amp-video/0.1/test/test-amp-video.js]: isn't it `noremoteplayback` ?

Comment (aghassemi) [reply extensions/amp-video/0.1/test/test-amp-video.js]: Yuu are right, source code (https://cs.chromium.org/chromium/src/third_party/WebKit/Source/core/html/media/HTMLMediaElementControlsList.cpp?q=noremote&sq=package:chromium&dr=CSs&l=15) says `noremoteplayback` this article (https://developers.google.com/web/updates/2017/03/chrome-58-media-updates) says `noremote` /cc @beaufortfrancois to update that article.

Comment (beaufortfrancois) [reply extensions/amp-video/0.1/test/test-amp-video.js]: Thank you so much @aghassemi! I'll update article now: https://github.com/google/WebFundamentals/pull/5042/

**Pull request merged**